File: doc/python/client/hello-world-async-wait.py
import hyperdex.client

c = hyperdex.client.Client("127.0.0.1", 1982)
p = c.async_put("kv", "some key", {"v": "Hello World!"})
p.wait()
print 'put "Hello World!"'
g = c.async_get("kv", "some key")
print 'got:', g.wait()

File: doc/ruby/client/hello-world-async-wait.rb
require "hyperdex"
c = HyperDex::Client::Client.new("127.0.0.1", 1982)
p = c.async_put(:kv, "some key", {:v => "Hello World!"})
p.wait()
puts 'put "Hello World!"'
g = c.async_get(:kv, "some key")
puts 'got:', g.wait()
